x/txreward/genesis.go

```diff
@@ -0,0 +1,21 @@
+package txreward
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+type GenesisState struct{}
```

Comment:
I think that it might be useful to include the initial pool deposits inside the `GenesisState`. This could be useful when: 

1. We want to set an initial amount. 
2. We want to export the state and later import it to upgrade the chain version. 

```suggestion
type GenesisState struct{
    PoolAmount sdk.Coins `json:"pool_amount"`
}
```